rest-api/src/main/kotlin/fund/cyber/markets/rest/configuration/AppContext.kt

```diff
@@ -0,0 +1,13 @@
+package fund.cyber.markets.rest.configuration
+
+import com.fasterxml.jackson.databind.ObjectMapper
+import fund.cyber.markets.cassandra.CassandraService
+
+object AppContext {
+
+    val jsonSerializer = ObjectMapper()
+
```

Comment:
replace common ObjectMapper with the instance based variable due to performance issue in multithreaded applications